modules/acquia_cms_image/composer.json

```diff
@@ -22,7 +22,11 @@
     },
     "extra": {
         "enable-patching": true,
-        "patches": {}
+        "patches": {
+            "drupal/imce": {
+                "3324281 - Property label with value Imce Link does not exist CKEditor5PluginDefinition": "https://www.drupal.org/files/issues/2022-11-30/3324281-7.patch"
+            }
+        }
```

Comment:
I think this patch is already getting applied in this PR - https://github.com/acquia/acquia_cms/pull/1331/files